docs/apache-spark-with-minio.md

```diff
@@ -21,7 +21,18 @@ Apache Spark is a fast and general engine for large-scale data processing. In th
 - Extract the `spark-2.1.2-bin-without-hadoop` tar ball in the directory where you're planning to install Spark.
 - Extract the `hadoop-2.8.2` tar ball in a separate directory. Copy the path to this directory.
 - Create a directory called `bin` inside the directory where `spark-2.1.2-bin-without-hadoop` was unzipped. Then move all the dependency jar files (downloaded in previous step) in this directory.
-- Open the file `$HADOOP_HOME/etc/hadoop/core-site.xml` for editing. Update the relevant fields as shown below. Note that in this example Minio server is running at `http://127.0.0.1:9000` with access key `minio` and secret key `minio123`.
+
+## 3. Start Spark-Shell
+
+Navigate to the directory where you extracted `spark-2.1.2-bin-without-hadoop` via terminal, and set the following environment variables
```

Comment:
* `s/via terminal//`  (terminal is not invoked in the past paragraphs)
* `s/$/:/`